glossary/es/open-standards/index.md

```diff
@@ -0,0 +1,7 @@
+---
+section: terms
+lang: es
+title: Estándares Abiertos
+---
+
+En general se entiende por estándares técnicos libres de restricciones de licencias. También puede ser interpretado en el sentido los estándares que se desarrollan de manera independiente del proveedor. Ver [Anonimización](/glossary/es/anonymisation/).
```

Comment:
The link to the term "anonimización" here should be *relative*. You need to change `/glossary/es/anonymisation/` to `../anonymisation/`.